modules/i3s/docs/api-reference/i3s-loader.md

```diff
@@ -159,15 +159,14 @@ const visibleTiles = tileset3d.tiles.filter(tile => tile.selected);
+| ------------------------------ | ---------------- | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `options.i3s.isTileset`        | `Bool` or `auto` | `auto`  | Whether to load `Tileset` (Layer 3D Index) file. If `auto`, will decide if follow `ArcGIS` tile layers' url convention                                                                                                                                                                                                          |
+| `options.i3s.isTileHeader`     | `Bool` or `auto` | `auto`  | Whether to load `TileHeader`(node) file. If `auto`, will decide if follow `argis` url convention                                                                                                                                                                                                                                |
+| `options.i3s.loadContent`      | `Bool`           | `true`  | Whether to load tile content (geometries, texture, etc.). Note: I3S dataset, each tile node has separate urls pointing to tile metadata and its actual tile payload. If `loadContent` is true, i3s loader will make a request to fetch the content fiile and decoded to the format as specified in [Tile Object](#tile-object). |
+| `options.i3s.tileset`          | `Object`         | `null`  | `Tileset` object loaded by I3SLoader or follow the data format specified in [Tileset Object](#tileset-object). It is required when loading i3s geometry content                                                                                                                                                                 |
+| `options.i3s.tile`             | `Object`         | `null`  | `Tile` object loaded by I3SLoader or follow the data format [Tile Object](#tile-object). It is required when loading i3s geometry content                                                                                                                                                                                       |
+| `options.i3s.useDracoGeometry` | `Bool`           | `false` | Use 'Draco' compressed geometry to show if applicable                                                                                                                                                                                                                                                                           |
```

Comment:
Don't we want this to be true to by default?

Reply:
Right